gpt_engineer/benchmark/types.py

```diff
@@ -72,5 +72,14 @@ class Benchmark:
 @dataclass
 class TaskResult:
     task_name: str
-    assertion_results: dict[str, bool]
+    assertion_results: List[dict[str, bool]]
     duration: float
+
+    # Returns success rate from 0.00 up to 1.00
+    @property
+    def success_rate(self) -> float:
+        succeeded = len(
+            [result for result in self.assertion_results if list(result.values())[0]]
+        )
+
+        return succeeded / len(self.assertion_results)
```

Comment:
I get: "ZeroDivisionError: division by zero" here when I run the first 5 problems.

Reply:
Has this been addressed? I think it happened because I ran few problems, only the first 5...
Or might it have been due to the catch clause that the list was empty?

Reply:
I don't manage to reproduce it, but I added a check in the beginning of the method:

```
if not self.assertion_results:
            return 0.0
```

Can you still catch it?